flexget/plugins/search/1337x.py

```diff
@@ -0,0 +1,165 @@
+        #infohash = infohash[:infohash.find(' ')]
+
+        magnetURL = str(soup.find("a", id="magnetdl").get('href')).lower()
+        torrentURL = str(soup.find("a", id="torrentdl").get('href')).lower()
+
+        # Can't update the title to the full title name because it breaks entry.remove
+        #title = str(soup.title.string).replace("Download Torrent ","").replace("| 1337x","")
+
+        entry['url'] = torrentURL
+        entry['urls'] = []
+        entry['urls'].append(torrentURL)
+        entry['urls'].append(magnetURL)
+
+    @plugin.internet(log)
+    def search(self, task, entry, config):
+        """
+            Search for entries on 1337x
+        """
+
+        if not isinstance(config, dict):
+            config = {}
+
+        order_by = ""
+        sorted = ""
+        if (isinstance(config.get('order_by'), str)):
+            if (config['order_by'] != "leechers"):
+                order_by = "/{0}/desc" . format(config['order_by'])
+                sorted = "sort-"
+
+        entries = set()
+
+        for search_string in entry.get('search_strings', [entry['title']]):
+
+            query = "{0}search/{1}{2}/1/" . format(sorted,quote(search_string.encode('utf8')), order_by)
+            log.debug('Using search params: {0}; ordering by: {1}' . format(search_string, order_by or "default"))
+            try:
+                page = self.get(self.base_url + query, None)
+                log.debug('requesting: %s', page.url)
+            except RequestException as e:
+                log.error('1337x request failed: %s', e)
+                continue
+
+            soup = get_soup(page.content)
+            if (soup.find('div', attrs={'class': 'tab-detail'}) != None):
+                for link in soup.find('div', attrs={'class': 'tab-detail'}).findAll('a',href=re.compile('^/torrent/')):
+
+                    li = link.parent.parent.parent
+
+                    title = str(link.text).replace("...","")
+                    infoUrl = self.base_url + str(link.get('href'))[1:]
+                    seeds = int(li.find("span", class_="green").string)
+                    leeches = int(li.find("span", class_="red").string)
+                    size = str(li.find("div", class_="coll-4").string)
+
+                    if size.split( )[1] == 'GB':
+                        size = int(float(size.split( )[0].replace(',', '')) * 1000 ** 3 / 1024 ** 2)
+                    elif size.split( )[1] == 'MB':
+                        size = int(float(size.split( )[0].replace(',', '')) * 1000 ** 2 / 1024 ** 2)
+                    elif size.split( )[1] == 'KB':
+                        size = int(float(size.split( )[0].replace(',', '')) * 1000 / 1024 ** 2)
+                    else:
+                        size = int(float(size.split( )[0].replace(',', '')) / 1024 ** 2)
+
+                    #print("Title: {0}; Seeds: {1}; Leech: {2}, Size: {3}" . format(title,seeds,leeches,size))
```

Comment:
Should remove this line or log it as debug or trace if you think it's necessary.